### PR TITLE
virsh_migrate_copy_storage: update invoking a function

### PR DESCRIPTION
**Pull request by dzhengfy**
Due to the function add_remote_disks_checking() API is change, the case need updates.

### CODE REVIEW

**Comment by dzhengfy** [comment]
Test on local host:
```
avocado run --vt-type libvirt virsh.migrate_copy_storage.normal_test.multi_disks.copy_storage_all.file_image
JOB ID     : d01eb475d6973f443d44334d9aa6b72c71154227
JOB LOG    : /root/avocado/job-results/job-2020-08-21T21.27-d01eb47/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.migrate_copy_storage.normal_test.multi_disks.copy_storage_all.file_image: PASS (158.69 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 161.90 s
```

**Comment by dzhengfy** [comment]
Depends on https://github.com/avocado-framework/avocado-vt/pull/2070
This PR should be merged together with above PR.

File: libvirt/tests/src/virsh_cmd/domain/virsh_migrate_copy_storage.py [hunk 1]
```diff
@@ -249,9 +250,10 @@ def run(test, params, env):
             logging.debug("Start migration...")
             cp_mig = copied_migration(test, vms, params)
             # Check the new disk can be working well with I/O after migration
-            utils_test.check_remote_vm_disks(vm, {'server_ip': remote_host,
-                                                  'server_user': remote_user,
-                                                  'server_pwd': remote_passwd})
+            vm_ip = vm.get_address()
```
Comment (Yingshun) [inline libvirt/tests/src/virsh_cmd/domain/virsh_migrate_copy_storage.py:253]: vm.get_address() without session will cause an issue to get migrated VM's IP sometimes, how about adding session into it?

Comment (dzhengfy) [reply libvirt/tests/src/virsh_cmd/domain/virsh_migrate_copy_storage.py:253]: I would like to use the IP on source host. Updated.

**Comment by dzhengfy** [comment]
Local test:
(1/1) type_specific.io-github-autotest-libvirt.virsh.migrate_options_shared.positive_test.pty_channel.with_postcopy: PASS (297.73 s)
(1/1) type_specific.io-github-autotest-libvirt.virsh.migrate_copy_storage.normal_test.multi_disks.copy_storage_all.file_image: PASS (142.44 s)
 (1/1) type_specific.io-github-autotest-libvirt.virsh.migrate_copy_storage.error_test.nonexist_image.copy_storage_all.file_image: PASS (126.18 s)

**Comment by dzhengfy** [comment]
@Yingshun Could you help review again?

File: libvirt/tests/src/virsh_cmd/domain/virsh_migrate_copy_storage.py [hunk 2]
```diff
@@ -178,9 +178,12 @@ def run(test, params, env):
...
                            vm.address_cache)
     vms = [vm]
-    if vm.is_dead():
-        vm.start()
-
+    vms_ip = {}
+    for vm in vms:
+        if vm.is_dead():
+            vm.start()
+        vm.wait_for_login()
```
Comment (Yingshun) [inline libvirt/tests/src/virsh_cmd/domain/virsh_migrate_copy_storage.py:185]: How about adding .close() to make sure vm sessions are closed?
I mean change this line to vm.wait_for_login().close().

Comment (dzhengfy) [reply libvirt/tests/src/virsh_cmd/domain/virsh_migrate_copy_storage.py]: Updated

**Comment by dzhengfy** [comment]
@Yingshun Please review again.

**Review by Yingshun — APPROVED** [review]

**Review by chunfuwen — APPROVED** [review]
LGTM

**Pull request merged**